canonical-web-and-design/jaas-dashboard
# Use Vanilla icons across the app

## Done

Update icons across the app

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check all icons load correctly

## Details

Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/453, 
Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1520
